### DISCUSSION TITLE
Per sip domain based view in Homer

### DISCUSSION THREAD

**Discussion by tite09-ops** [sipcapture/homer#668] [Q&A] [answered]
Hello, i'm using homer 7. I have many users who use my homer web interface and i want to filter view for each user in order to be sure that each user see only calls related to his sip domain. Is Homer able to do that ? if yes how can we do that ? thank you

**Comment by adubovikov** [comment, marked as answer]
Hi,

Yes, this is possible in Homer 7 via **group isolation** in homer-app config.

Use `group_settings.isolate_groups` and `isolate_query` (or `isolate_queries` for multiple groups). For users in an isolated group, every search gets an extra SQL filter.

Example for one SIP domain:

```json
"group_settings": {
  "isolate_groups": ["tenant_example"],
  "isolate_queries": [
    "(from_domain = 'example.com' OR to_domain = 'example.com')"
  ]
}
```

Create users with `usergroup: "tenant_example"`. They will only see calls matching that filter. Admin users stay outside `isolate_groups` and see everything.

For several domains, add one group per domain and one entry in `isolate_queries` per group.

This is config-driven, not automatic from the user profile. You assign the group manually (or via LDAP/OAuth group mapping if you use that).
